CHANGELOG.md

```diff
@@ -2,11 +2,15 @@
 
 ## XX.XX.XX - 2022-XX-XX
 
+### PaymentSheet
+
+* [ADDED][5769](https://github.com/stripe/stripe-android/pull/5769) Added support for showing payment methods that require a shipping address via `PaymentSheet.Configuration.allowsPaymentMethodsRequiringShippingAddress`. If set to `true`, PaymentSheet displays these payment methods even if no `shippingDetails` are passed in.
```

Comment:
Can we include something like the last two sentences of the iOS CHANGELOG: 

The iOS changelog, for reference:
* [Added] Added PaymentSheet.Configuration.allowsPaymentMethodsRequiringShippingAddress. Previously, to allow payment methods that require a shipping address (e.g. Afterpay and Affirm) in PaymentSheet, you attached a shipping address to the PaymentIntent before initializing PaymentSheet. Now, you can instead set this property to true and set PaymentSheet.Configuration.shippingDetails to a closure that returns your customers' shipping address. The shipping address will be attached to the PaymentIntent when the customer completes the checkout.

i.e. instruction to also set `PaymentSheet.Configuration.shippingDetails` or whatever the Android equivalent is, and the part about where the shipping address ends up.

Reply:
Rebased this pull request onto `address-element-private-beta` (which adds `FlowController.shippingDetails`, the Android equivalent of iOS’s `shippingDetails` closure) and updated the wording. Let me know what you think!